api/src/main/java/org/pmiops/workbench/google/CloudStorageService.java

```diff
@@ -9,5 +11,6 @@
 
   public String readInvitationKey();
   public String readBlockscoreApiKey();
+  public List<Blob> getBlobList(String bucketName, String directory);
```

Comment:
FYI: I've changed this in my PR to the more generic `BlobInfo` because `Blob` seems to be more challenging to fake out for testing. Again - happy to do that change in my PR, but thought I'd mention it here in case you hit the same issue.

Reply:
Fixed